### ISSUE TITLE
Jqxtreegrid.js v3.4.0 and this version of JQXTreegrid is not accessibility compliant(Section 508 Compliant)

### ISSUE THREAD

**Issue by Raunak0786** [jqwidgets/jQWidgets#195]
We are using Jqxtreegrid.js v3.4.0 and this version of JQXTreegrid is not accessibility compliant(Section 508 Compliant) 
Below accessibility features Not including in JQXTreeGrid:
1. Table should be accessible using Screen readers like “Narrator”, “Jaws” and read all cell with column headers ,
2. ‘T’ command should work in narrator scan mode to let users know available tables on the page
Read proper column and rows count.
3. This table is also not accessible using screen reader like “Narrator” with edge, Jaws with chrome

**Comment by jqwidgets** [comment]
Section 508 Compliant is not supported by our product.

**Comment by Raunak0786** [comment]
Hi Team,

Do you have any plan to make it Compliant  ?
Please let us know if there is any updated version of JQXTreegrid available that have accessibility features incorporated. 
For example , following accessibility   ##features:
1.       Table headers should be sortable using  #keyboard.
2.       Table pagination controls should be accessible using keyboard.
3.       Table is accessible using Screen readers like “Narrator”, “Jaws” .

Thanks
Raunak Deep

**Comment by jqwidgets** [comment]
Raunak Deep, the item is with low priority. In case this feature is very important for your company, please contact sales@jqwidgets.com with details about it and we can create it as a custom development  task
